packages/liferay-npm-scripts/src/config/jest.config.js

```diff
@@ -7,17 +7,17 @@
 const path = require('path');
 
 module.exports = {
+	coverageDirectory: 'build/coverage',
 	globals: {
 		Liferay: {}
 	},
-	coverageDirectory: 'build/coverage',
 	modulePathIgnorePatterns: ['/__fixtures__/', '/build/', '/classes/'],
 	testMatch: ['**/test/**/*.js'],
 	testPathIgnorePatterns: ['/node_modules/', '<rootDir>/test/stories/'],
 	testResultsProcessor: 'liferay-jest-junit-reporter',
 	testURL: 'http://localhost',
 	transform: {
-		'\\.soy$': path.join(__dirname, '..', 'jest', 'transformSoy.js'),
-		'.+': path.join(__dirname, '..', 'jest', 'transformBabel.js')
+		'.+': path.join(__dirname, '..', 'jest', 'transformBabel.js'),
+		'\\.soy$': path.join(__dirname, '..', 'jest', 'transformSoy.js')
```

Comment:
Hm. I am not sure about this one. I think it might use the "first match wins" approach, which means that before this change, `transformSoy.js` would have applied to ".soy" files, and `transformBabel.js` to everything else. After this change, it looks like the opposite.

Reply:
You're probably right... we can err on the side of cauting here and just ignore sorting here and in `NORMALIZERS` as well

Reply:
Pushed an additional commit restoring the plugin order and disabling eslint for those lines